bears/documentation/DocumentationStyleBear.py

```diff
@@ -134,6 +134,11 @@ def run(self, filename, file, language: str,
                 new_metadata, doc_comment.docstyle_definition,
                 doc_comment.marker, doc_comment.indent, doc_comment.position)
 
+            if file[doc_comment.range.start.line - 2] != '\n':
```

Comment:
I note that we already had this 'chat' about not using literal values at the end of the last phase, regarding indent size.  Not happy that you need to be told again to design for multiple languages.